Security/verify_IPS_from_the_command_line.md

```diff
@@ -0,0 +1,158 @@
+{{{
+  "title": "Verify IPS Install from the Command Line",
+  "date": "09-15-2015",
+  "author": "Matthew Close",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": false
+}}}
+
+### Verify IPS Install from the Command Line
+
+#### Overview
+
+This tutorial will show how to trigger a CenturyLink Cloud IPS rule and view the notification using webhooks.
+
+#### Requirements
+
++ A CenturyLink Cloud host running the IPS service. To get started with IPS, please follow the instructions in [Getting Started with Intrusion Prevention System (IPS)](../Security/getting-started-with-ips.md)
++ ruby
++ sinatra
++ ngrok
++ curl
+
+**Note:** It is possible to complete this test without installing ruby, sinatra, and ngrok by using other HTTP request inspectors. For example, you could use [requestb.in](http://requestb.in). If you choose to use another inspector, you can skip to the second section. However, please be sure to use the URL provided by [requestb.in](http://requestb.in) instead of the ngrok URL.
+
+#### Instructions for setting up your webhook listener
```

Comment:
can we simplify this at all?  there are ample free services so customers don't have to setup a whole webhook listener just to verify the service is functional.  zapier.com for example could be had in 2 minutes and a url to listen on.  just seems like quite a bit of technical work for a test event.